metadata.json

```diff
@@ -1,2 +1,8 @@
 [
+{
+    "name": "brave-fix",
+    "aliases": ["bf"],
+    "kind": {"mime":"application/javascript"},
+    "resourcePath": "resources/brave-fix.js"
```

Comment:
You still need it to be:
```json
    "resourcePath": "brave-fix.js"
```

But add the `resources/brave-fix.js` file into this PR as well (move it from https://github.com/brave/adblock-lists/blob/master/brave-fix.js)